packages/pictograms/pictograms.yml

```diff
@@ -2843,32 +2816,30 @@
     - precipitation
     - forecast
 - name: hamburg--philharmone
-  friendly_name: Hamburg Elbphilharmonie
+  friendly_name: Hamburg Philharmone
   aliases:
-    - Hamburg Elbphilharmonie
+    - Hamburg Philharmone
     - Hamburg
     - Germany
-    - Elbphilharmonie
+    - Philharmone
     - city
     - cities
 - name: handicap
-  friendly_name: Disabled user
+  friendly_name: Handicap
```

Comment:
I think we should avoid using this term. It's outdated and generally frowned upon. Unicode calls these symbols "Wheelchair symbol", though it's technically moreso a riff of the [International Symbol of Access](https://en.wikipedia.org/wiki/International_Symbol_of_Access). That would be too cumbersome of a name though imo. The previous name `Disabled user` feels much more appropriate

Reply:
Inadvertently pulled in older code